flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyClientServerSslTest.java

```diff
@@ -65,6 +68,60 @@ public void testValidSslConnection() throws Exception {
 
 		Channel ch = NettyTestUtil.connect(serverAndClient);
 
+		SslHandler sslHandler = (SslHandler) ch.pipeline().get("ssl");
+		assertTrue("default value should not be propagated", sslHandler.getHandshakeTimeoutMillis() >= 0);
+		assertTrue("default value should not be propagated", sslHandler.getCloseNotifyTimeoutMillis() >= 0);
+
+		// should be able to send text data
+		ch.pipeline().addLast(new StringDecoder()).addLast(new StringEncoder());
+		assertTrue(ch.writeAndFlush("test").await().isSuccess());
+
+		NettyTestUtil.shutdown(serverAndClient);
+	}
+
+	/**
+	 * Verify valid (advanced) ssl configuration and connection.
+	 */
+	@Test
+	public void testValidSslConnectionAdvanced() throws Exception {
```

Comment:
please deduplicate code with `testValidSslConnection`

Reply:
This hasn't been addressed. Those tests differ only with expected values and passed `NettyConfig`